dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -243,7 +243,14 @@ public Item createTemplateItem(Context context, Collection collection) throws SQ
     {
         return itemDAO.findAll(context, true, true, true, since);
     }
-
+	
+	@Override
+    public Iterator<Item> findInArchiveOrWithdrawnNonDiscoverableModifiedSince(Context context, Date since)
```

Comment:
It'd be nice to add a Unit Test for this new method into 
https://github.com/DSpace/DSpace/blob/master/dspace-api/src/test/java/org/dspace/content/ItemTest.java

Admittedly, our test coverage for `ItemService` is lacking right now, but we're trying to improve that. This could be an opportunity to at least ensure we are testing new methods.

Reply:
I will look into that. If there is a test to its "sister" method, it should be mainly c&p